cpp/tensorrt_llm/kernels/groupRmsNormKernels/groupRmsNormKernels.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2022-2025, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+#include <NvInferRuntime.h>
+#include <cuda_bf16.h>
+#include <cuda_fp16.h>
+
+#include "tensorrt_llm/common/assert.h"
+#include "tensorrt_llm/common/cudaUtils.h"
+
+namespace tensorrt_llm::kernels::group_rms_norm
+{
+
+template </*number of inputs*/ int n>
+struct GroupRMSParams
+{
+    float4* inputs[n];
+    float4* outputs[n];
+    float4 const* weights[n];
+    int input_dims[n];
```

Comment:
```suggestion
    int input_dims[n];
```
The name is a bit confusing, because it contains the last dim of all the inputs

Reply:
Renamed to input_last_dims.